### PR TITLE
Describe what's next in unstyled

### PR DESCRIPTION
**Pull request by michaldudak**
Adds some insights on our plans regarding unstyled/base

### CODE REVIEW

File: docs/pages/blog/mui-core-v5.md [hunk 1]
```diff
@@ -531,8 +531,23 @@ It's new, link core and x roadmaps.
 
 ### Core
 
-- Unstyled
-- Joy
+#### Unstyled components and hooks
+
+We still have a lot of work to do to have a great unstyled package.
```
Comment (oliviertassinari) [inline docs/pages/blog/mui-core-v5.md]: Maybe to emphasize on the end product, more than how it's delivered?

```suggestion
We still have a lot of work to do to have a great set of unstyled components.
```

File: docs/pages/blog/mui-core-v5.md [hunk 2]
```diff
@@ -531,8 +531,23 @@ It's new, link core and x roadmaps.
...
+You can keep track of our progress in [#27170](https://github.com/mui-org/material-ui/issues/27170).
+
+Our high-level plan is to use the unstyled components and hooks as a base of Material components.
+We are aiming to complete this effort with the next major release (v6).
+As of now, you can evaluate the unstyled primitives in the `@mui/base` package or check out the new implementation of Material components in `@mui/material-next` package.
```
Comment (oliviertassinari) [inline docs/pages/blog/mui-core-v5.md]:
```suggestion
As of now, you can evaluate the unstyled primitives in the `@mui/base` package or check out the next implementation of the Material Design components in the `@mui/material-next` package (targeted at v6).
```

File: docs/pages/blog/mui-core-v5.md [hunk 3]
```diff
@@ -531,8 +531,23 @@ It's new, link core and x roadmaps.
...
+Please note that both packages are in alpha and breaking changes can and will occur.
```
Comment (oliviertassinari) [inline docs/pages/blog/mui-core-v5.md]: I think that we can emphasize being in alpha so we can do breaking changes, it's likely already enough to be used in production. 

```suggestion
Please note that both packages are in an alpha state so we can release breaking changes.
```

File: docs/pages/blog/mui-core-v5.md [hunk 4]
```diff
@@ -531,8 +531,23 @@ It's new, link core and x roadmaps.
...
+We want to take the opportunity to create the best APIs we possibly can.
+
+You can help us shape these new packages by taking part in discussions.
+There are [RFCs](https://github.com/mui-org/material-ui/issues?q=is%3Aopen+label%3Adiscussion+%5BRFC%5D) waiting for responses.
+Do not hesitate to let us know what you think.
+We highly value the voice of the community.
```
Comment (oliviertassinari) [inline docs/pages/blog/mui-core-v5.md]:
```suggestion
We highly value the voice of the community, this package is built to serve you.
```

**Review by oliviertassinari — COMMENTED** [review]
Feel free to push directly on my branch. It seems simpler

**Comment by michaldudak** [comment]
All right, I applied your suggestions and pushed directly.

**Comment by oliviertassinari** [comment]
thanks

**Pull request closed without merging**